x-pack/plugins/index_management/public/application/components/template_form/steps/step_review.tsx

```diff
@@ -186,6 +187,21 @@ export const StepReview: React.FunctionComponent<Props> = React.memo(
                 {version ? version : <NoneDescriptionText />}
               </EuiDescriptionListDescription>
 
+              {/* Allow auto create */}
+              {isLegacy === false && (
```

Comment:
I think here as well, the variable `isLegacy` might be undefined, for example when editing an existing "non-legacy" template, the property is not displayed in the review step.